client/src/components/StreamField/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createStore, applyMiddleware } from 'redux';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+import { StreamField, streamFieldReducer } from 'react-streamfield';
+
+const store = createStore(streamFieldReducer, applyMiddleware(thunk));
+
+const init = (name, options, currentScript) => {
+  // document.currentScript is not available in IE11. Use a fallback instead.
+  const context = currentScript ? currentScript.parentNode : document.body;
+  // If the field is not in the current context, look for it in the whole body.
+  // Fallback for sequence.js jQuery eval-ed scripts running in document.head.
+  const selector = `[name="${name}"]`;
+  const field = (context.querySelector(selector)
+                 || document.body.querySelector(selector));
```

Comment:
Could we select the field based on ID instead? That's the standard way we attach Javascript behaviour to widgets (see https://github.com/wagtail/wagtail/blob/master/wagtail/admin/templates/wagtailadmin/widgets/draftail_rich_text_area.html for example).